draft-packages/select/KaizenDraft/Select/styles.elm.scss

```diff
@@ -219,3 +239,17 @@
     background-color: transparent;
   }
 }
+
+.autosizeInputContainer {
+  @include kz-typography-paragraph-body;
+  @include ca-inherit-baseline;
```

Comment:
These includes fix a long standing bug where the search text font would be the browser default (a sans serif font). I fixed it at the same time, because I needed to make sure the vertical alignment was correct.